BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.kt

```diff
@@ -8,11 +8,12 @@ import androidx.room.RoomDatabase
 
 // Ref: https://developer.android.com/training/data-storage/room/migrating-db-versions
 @Database(
-        version = 3,
+        version = 4,
         entities = [AnalyticsEvent::class],
         autoMigrations = [
             AutoMigration(from = 1, to = 3),
```

Comment:
```suggestion
            AutoMigration(from = 1, to = 2),
```
I noticed this may be a pre-existing bug. The migrations are supposed to [count up in incremental pairs](https://medium.com/androiddevelopers/room-auto-migrations-d5370b0ca6eb).

Reply:
so it should be 1->2, 2->3, 3->4?

Reply:
Yup yeah that's it.